incident.go

```diff
@@ -87,6 +96,8 @@ type Incident struct {
 	Body                 IncidentBody         `json:"body,omitempty"`
 	IsMergeable          bool                 `json:"is_mergeable,omitempty"`
 	ConferenceBridge     *ConferenceBridge    `json:"conference_bridge,omitempty"`
+	AssignedVia          string               `json:"assigned_via,omitempty"`
+	Occurrence           Occurrence           `json:"occurrence,omitempty"`
```

Comment:
Does this always come back for an incident, or only some of the time. If it's only some of the time, might it make sense for this to be a pointer type (like `*ConferenceBridge`)?

Reply:
Looking at the API doc (which doesn't have this field in the example) it makes a lot of sense to me, thanks for pointing this out. 26b8fa2